src/libasr/codegen/wasm_to_x86.cpp

```diff
@@ -68,6 +69,27 @@ class X86Visitor : public WASMDecoder<X86Visitor>,
         }
     }
 
+    void visit_EmtpyBlockType() {}
+
+    void visit_If() {
+        unique_id.push_back(offset);
+        m_a.asm_pop_r32(X86Reg::eax);
+        m_a.asm_cmp_r32_imm8(LFortran::X86Reg::eax, 1);
+        m_a.asm_je_label(".then_" + std::to_string(unique_id.back()));
+        m_a.asm_jmp_label(".else_" + std::to_string(unique_id.back()));
+        m_a.add_label(".then_" + std::to_string(unique_id.back()));
```

Comment:
For later (but probably quite soon): we should not be using strings for labels, but just unique integers. The unique integer can just be a counter in this visitor, just an integer, see above.

Reply:
(we might need to have a stack of unique ids...)